src/store/drawing/__tests__/actions-test.ts

```diff
@@ -0,0 +1,24 @@
+import { annotations } from '../../../drawing/__mocks__/drawingData';
+import { addDrawingPathGroupAction } from '../actions';
+
+const {
+    target: { path_groups: pathGroups },
```

Comment:
Should we export these path groups separately to avoid this destructuring in each test suite?